configure.ac

```diff
@@ -195,6 +180,27 @@ AC_HELP_STRING([--with-sdk-install-path=DIR path to external libs and headers],
 AC_SUBST(SDK_INSTALL_PATH)
 
 ##########################################################################
+# Run platform specific checks and settings
+##########################################################################
+IMPLEMENTATION_NAME=""
```

Comment:
I thought it was discussed in the cloud call saying we don't need to worry about this flag as it only affects dpdk wrapper.